src/ui-lib/sass/00-settings/gravy.scss

```diff
@@ -1,2 +1,22 @@
 @import 'gravy';
-@include gravy-init();
+
+$gravy: (
+  typefaces: (
+    primary: (
+      stack: ('Roboto', sans-serif),
+      weights: (
+        regular: 400,
+        bold: 'bold'
+      )
+    ),
+    secondary: (
+      stack: ('Cabin', sans-serif),
+      weights: (
+        regular: 400,
+        bold: 600
+      )
+    )
+  )
+);
+
+@include gravy-init;
```

Comment:
Just an idea, but for developer convenience, it might be good to add a comment here with the URL of any Gravy docs that cover the set of vars/maps/mixins it generates at this point.